code/CMSProfileController.php

```diff
@@ -20,6 +20,9 @@ class CMSProfileController extends LeftAndMain
 
     private static $required_permission_codes = 'CMS_ACCESS';
 
+    /**
+     * @deprecated 5.4.0 Will be renamed model_class
```

Comment:
```suggestion
     * @deprecated 5.4.0 Will be renamed to $model_class
```
Update here and in all other files in all all the PRs

Reply:
Done